test/unit/server-statefulset.bats

```diff
@@ -383,7 +383,7 @@ load _helpers
 @test "server/StatefulSet: gossip encryption disabled in server StatefulSet when servers are disabled" {
   cd `chart_dir`
   local actual=$(helm template \
-      -x templates/enterprise-license.yaml  \
+      -x templates/enterprise-license-job.yaml  \
```

Comment:
Nice catch. Actually that test doesn't make any sense because it's looking for certain keys within the server statefulset when servers are disabled. When servers are disabled then the statefulset isn't rendered at all so I just deleted the test.